packages/js/src/server/integrations/unhandled_rejection_plugin.ts

```diff
@@ -2,16 +2,17 @@ import { Types } from '@honeybadger-io/core'
 import Client from '../../server'
 import UnhandledRejectionMonitor from './unhandled_rejection_monitor'
 
+const unhandledRejectionMonitor = new UnhandledRejectionMonitor()
+
 export default function (): Types.Plugin {
   return {
     load: (client: typeof Client) => {
-      if (!client.config.enableUnhandledRejection) {
-        return
-      }
-      const unhandledRejectionMonitor = new UnhandledRejectionMonitor(client)
-      process.on('unhandledRejection', function honeybadgerUnhandledRejectionListener(reason, _promise) {
-        unhandledRejectionMonitor.handleUnhandledRejection(reason, _promise)
-      })
+      unhandledRejectionMonitor.setClient(client)
```

Comment:
I can't remember... is there any concern of [multiple clients overriding this](https://docs.honeybadger.io/lib/javascript/guides/working-with-multiple-clients/)? I believe we only load these plugins for the singleton, so maybe it's not a current concern? I'm just wondering if there could ever be a case where someone might want to manually add a plugin.

Reply:
Yep, that's all right. Only the singleton client gets our plugins, since many of them are global. I think this is just something we'll need to think about if we ever decide to expose the plugin system and/or plugins to end users (Sentry does that, iirc). I think it's OK to assume there will ever only be one instance of each plugin currently.

Reply:
> I think as a user, if I used a method called factory, I'd expect to get a duplicate with the same functionality... since users didn't add these plugins in the first place, I don't think they'd expect to have to add them after using the factory.

Do we all agree that this should be the expected behavior? If so, we should make sure that plugins can be loaded in more than one client simultaneously, or even come up with singletons of them if needed.

When we have this sorted out, we can update the `factory` method to load plugins by default for all client instances.

What do you think?

Reply:
I created a new issue for us to be able to continue this discussion! In the meanwhile, this PR assumes the plugin is loaded in one client only, which I think is fine for now. 

https://github.com/honeybadger-io/honeybadger-js/issues/1181

Reply:
Thanks for creating the issue!